pyproject.toml

```diff
@@ -9,7 +9,6 @@ authors = [
     { name = "Zhang Yunjun", email = "yunjunz@outlook.com" },
     { name = "Heresh Fattahi", email = "hersh.fattahi@gmail.com" },
 ]
-readme = { file = "README.md", content-type = "text/markdown" }
```

Comment:
@yunjunz  I think you might just need to put `docs/README.md`? I hadn't noticed that the README isn't in the top level directory

Reply:
ah, I guess that's why it's not working yet! Thank you @scottstanie!

Reply:
It works now: https://test.pypi.org/project/mintpy/. Cheers!